### PR TITLE
Created adapter for timedelta and added tests related to new adapter

### PR DESCRIPTION
**Pull request by AlexTorx**
I noticed there was no adapter for datetime.timedelta json rendering object.

I created one using the timedelta.total_seconds method from datetime.timedelta object as a serialization method.

### CODE REVIEW

**Comment by jssuzanne** [comment]
great. Why second only ? Maibe a parameter to select between second, micro-second or other unit should be fun if we want other than second

**Comment by AlexTorx** [comment]
I have added a mode parameter to select how timedelta objects should be serialized (microseconds, milliseconds, seconds, minutes, hours, days or weeks).

Tests now cover these changes.

Updated copyrights

File: anyblok_pyramid/adapter.py [hunk 1]
```diff
@@ -49,10 +49,66 @@ def timedelta_adapter(obj, request):
...
+    The mode parameter can be used to set the output of the adpater.
+    This parameter can either be :
+
+        * microseconds
+        * milliseconds
+        * seconds (default)
+        * minutes
+        * hours
+        * days
+        * weeks
```
Comment (jssuzanne) [inline anyblok_pyramid/adapter.py]: You should create constante or enum to help

File: anyblok_pyramid/tests/test_adapter.py [hunk 2]
```diff
@@ -39,6 +42,59 @@ def add_route_and_views(config):
...
+
+        def add_route_and_views(config, mode='seconds'):
+            config.add_route('dbname', '/test/')
+            config.add_view(
+                get_timedelta, route_name='dbname', renderer='json')
+            json_renderer = JSON()
+            json_renderer.add_adapter(
+                timedelta, lambda obj, request: timedelta_adapter(
+                    obj, request, mode))
+            config.add_renderer('json', json_renderer)
```
Comment (jssuzanne) [inline anyblok_pyramid/tests/test_adapter.py]: it's complicated to understand the use with a lamda.

Maybe the better action is to decorate the timedelta_adapter

```
def timedelta_factory(mode=SECONDS):
    def timedelta_adapter(obj, request):
          # copy your adapter here

    return wrap
```

so the configuration is easer

```
from anyblok_pyramid.adapter import timedelta_factory, MILLISECONDS

...

json_renderer.add_adapter(timedelta, timedelta_factory(MILLISECONDS))
```

**Comment by jssuzanne** [comment]
> I have added a mode parameter to select how timedelta objects should be serialized (microseconds, milliseconds, seconds, minutes, hours, days or weeks).
> 
> Tests now cover these changes.
> 
> Updated copyrights

You miss to add notification in the change log

File: anyblok_pyramid/adapter.py [hunk 3]
```diff
@@ -40,65 +42,88 @@ def datetime_adapter(obj, request):
...
+            json_renderer.add_adapter(
+                timedelta, timedelta_adapter_factory(TimeDeltaModes.HOURS))
+            config.add_renderer('json', json_renderer)
+
+        :param obj: timedelta obj
+        :param mode: str
+        :rtype: str, seconds corresponding to timedelta
+        """
+
+        import ipdb; ipdb.set_trace()
```
Comment (jssuzanne) [inline anyblok_pyramid/adapter.py]: Oups !!!

**Comment by AlexTorx** [comment]
> Great, thank for this PR, I let you merge it

Yeah that would be really cool, but I guess I do not have any right in order to perform that...

**Pull request merged**